frontend/config/dev.env.js

```diff
@@ -3,7 +3,10 @@ const merge = require('webpack-merge')
 const prodEnv = require('./prod.env')
 
 module.exports = merge(prodEnv, {
-  NODE_ENV: '"development"',
-  MAPBOX_TOKEN: '"'+process.env.MAPBOX_TOKEN+'"',
-  MAPBOX_STYLE: '"'+process.env.MAPBOX_STYLE+'"'
-})
\ No newline at end of file
+    NODE_ENV: '"development"',
```

Comment:
no need for changes here
merge(prodEnv, {}), config will be copied from .prod.env